distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/AbstractIntegrationTest.java

```diff
@@ -397,6 +397,10 @@ protected UrlResourceReaderConfigurator getUrlResourceReaderConfigurator() {
 
     protected Option[] configureAdditionalBundles() {
         return options(junitBundles(),
+                wrappedBundle(mavenBundle("ddf.catalog.registry",
```

Comment:
This should not be in the itests - if you need to interact with the catalogframework directly requiring something than what is already available then you need to create a bundle in the SDK app to do so.